README.md

```diff
@@ -10,69 +10,81 @@
 
 [![Built with Astro](https://astro.badg.es/v2/built-with-astro/small.svg)](https://astro.build)
 
-## 🏃 Get Started
+This is the front-end repo for [YoungVision.org](https://www.youngvision.org/) ([CMS Repo](https://github.com/YoungVision-eV/website-cms)), built using [Astro](https://astro.build).
 
-### 📦 Requirements
+## 📦 Prerequisites
 
 - Node.js v16 or higher
 - [pnpm](https://pnpm.io)
 - [better-commits](https://github.com/Everduin94/better-commits)
 
-### 📦 Install
+## 📥 Installation
+
+Clone the repository and install dependencies:
 
 ```bash
+git clone https://github.com/YoungVision-eV/website.git
+cd website
 pnpm install
 ```
 
-### 📦 Working on tasks
+## 🚧 Working on Tasks
 
-Choose an issue and then create a branch with
+1. Choose an issue to work on.
+2. Create a new branch using:
 
-```bash
-better-branch
-```
+   ```bash
+   better-branch
+   ```
 
-After you've done some work on it commit your changes with
+3. After making changes, commit them using:
 
-```bash
-better-commits
-```
+   ```bash
+   better-commits
+   ```
 
-## 🚀 Project Structure
+4. Push your branch and open a Pull Request (PR) following the process below.
 
-Inside of your Astro project, you'll see the following folders and files:
+## 📁 Project Structure
 
 ```text
 /
-├── public/
+├── public/            # Static assets (images, icons, etc.)
 ├── src/
-│   └── pages/
-│       └── index.astro
-└── package.json
+│   ├── components/    # UI components
+│   ├── pages/         # Astro pages
+│   ├── styles/        # Global styles
+│   └── utils/         # Utility functions
+├── package.json       # Project dependencies and scripts
+└── astro.config.mjs   # Astro configuration
 ```
 
-Astro looks for `.astro` or `.md` files in the `src/pages/` directory. Each page is exposed as a route based on its file name.
-
-There's nothing special about `src/components/`, but that's where we like to put any Astro/React/Vue/Svelte/Preact components.
+## 🛠️ Commands
 
-Any static assets, like images, can be placed in the `public/` directory.
+| Command                    | Description                               |
+| -------------------------- | ----------------------------------------- |
+| `pnpm install`             | Installs dependencies                     |
+| `pnpm run dev`             | Starts a local development server         |
+| `pnpm run build`           | Builds the production site                |
+| `pnpm run preview`         | Previews the built site before deployment |
+| `pnpm run astro ...`       | Runs Astro CLI commands                   |
+| `pnpm run astro -- --help` | Get help using the Astro CLI              |
```

Comment:
This is missing some scripts:
- `pnpm run lint`
- `pnpm run format`
- `pnpm run test`